modules/package.json

```diff
@@ -0,0 +1,74 @@
+{
+    "name": "ilib-sample",
+    "version": "1.0.0",
+    "main": "./src/index.js",
+    "description": "Sample application that demostrates how to use the ilib-localeinfo module",
+    "keywords": [
+        "internationalization",
+        "i18n",
+        "localization",
+        "l10n",
+        "globalization",
+        "g11n",
+        "date",
+        "time",
+        "format",
+        "locale",
+        "translation"
+    ],
+    "homepage": "https://github.com/iLib-js/ilib-sample",
+    "bugs": "https://github.com/iLib-js/ilib-sample/issues",
+    "email": "marketing@translationcircle.com",
+    "license": "Apache-2.0",
+    "author": {
+        "name": "Edwin Hoogerbeets",
+        "web": "http://www.translationcircle.com/",
+        "email": "edwin@translationcircle.com"
+    },
+    "contributors": [
+        {
+            "name": "Edwin Hoogerbeets",
+            "email": "ehoogerbeets@gmail.com"
+        },
+        {
+            "name": "Goun Lee",
+            "email": "goun.lee@lge.com"
+        }
+    ],
+    "files": [
+        "src",
+        "README.md",
+        "LICENSE"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "git@github.com:iLib-js/iLib-sample.git"
+    },
+    "scripts": {
+        "build": "npm-run-all build:assemble build:webtest",
+        "build:assemble": "ilib-assemble --localefile locales.json assembled src",
+        "build:webtest": "webpack-cli --env dev --config webpack.config.js",
+        "test:web": "npm-run-all build:assemble build:webtest ; open-cli ./module.html ; open-cli ./module.html -- firefox",
+        "test": "node src/index.js",
```

Comment:
I think `npm run test` command is not valid here. isn't it ok to be removed to avoid confusion?

Reply:
It is valid, actually. It is mentioned in the README. Try:

`npm run test ko-KR`

Reply:
I faced an error below
```
goun@goun-linux:~/Source/ilib-samples/modules$ npm run test ko-KR

> ilib-sample@1.0.0 test /home/goun/Source/ilib-samples/modules
> node src/index.js "ko-KR"

file:///home/goun/Source/ilib-samples/modules/node_modules/ilib-localeinfo/src/index.js:141
    static {
           ^

SyntaxError: Unexpected token '{'
    at Loader.moduleStrategy (internal/modules/esm/translators.js:145:18)
npm ERR! code ELIFECYCLE

```

Reply:
You need node v16 or later

Reply:
When I test with node v16.0 it shows and same error. but the version v16.11 was ok.
I think we'd better specify the working minimum node version in package.json.